compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -1316,6 +1319,10 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
 
     serviceFiles ++ enumFiles ++ messageFiles :+ fileDescriptorObjectFile
   }
+
+  private def getAnnotationList(message: Descriptor) = {
+    message.getOptions.getExtension[MessageOptions](Scalapb.message).getAnnotationsList().asScala.toSeq
```

Comment:
For consistency with all other options: please move this function to `MessageDescriptorPimps` in `DescriptorPimps.scala`. Rename it to `annotationsOption` and use the `scalaOptions` member to get the extension.